examples/applications/plot_out_of_core_classification.py

```diff
@@ -70,6 +72,9 @@ def __init__(self, encoding='latin-1'):
 
     def handle_starttag(self, tag, attrs):
         method = 'start_' + tag
+        for attr in attrs:
```

Comment:
Surely this only applies to a single tag name, not all, and can be handled in the appropriate handle_* method

Reply:
Ping.